infra/nginx/nginx.conf

```diff
@@ -0,0 +1,35 @@
+user nginx;
+worker_processes  1;
```

Comment:
Why only one worker process? I'm not familiar with Nginx tuning.

Reply:
This is the number of operating system threads it will use. Recommended is to set to number of cpu's. They've added an `auto` option for this, so we should use that.
https://nginx.org/en/docs/ngx_core_module.html#worker_processes